subcommand/controller/command.go

```diff
@@ -93,10 +96,12 @@ func (c *Command) Run(_ []string) int {
 	}
 
 	if os.Getenv("ENABLE_WEBHOOKS") != "false" {
```

Comment:
I agree. I think we should use flags for everything!! 😄 

Additionally, I think we should always deploy our webhooks, so maybe it doesn't make sense to have this check at all, and always deploy the webhook. Do you think that makes sense.

Reply:
This comes from the generated code because when running locally webhooks don't work (since kube can't call into your laptop. I think the makefile actually used to set this env var for you when running locally but maybe we've lost that target.

Down to switch to a flag though, separate PR makes sense I think.

Reply:
In that case, i'll add a task to address this for now and resolve this here.